src/erlcloud_cloudwatch_logs.erl

```diff
@@ -355,6 +428,8 @@ maybe_cw_request({ok, Config}, Action, Params) ->
 maybe_cw_request({error, _} = Error, _Action, _Params) ->
     Error.
 
+maybe_json({ok, <<>>}) ->
+    ok;
```

Comment:
Is it actually needed?
This may result in `case_clause`s in various places, like [here](https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_cloudwatch_logs.erl#L233)

Reply:
yes since jsx fails on empty responses. 
but corrected to not fail rest